inst/include/armadillo_bits/arma_version.hpp

```diff
@@ -23,7 +23,7 @@
 
 #define ARMA_VERSION_MAJOR 14
 #define ARMA_VERSION_MINOR 2
-#define ARMA_VERSION_PATCH 1
+#define ARMA_VERSION_PATCH 2
```

Comment:
@eddelbuettel Not quite yet.  I've reverted this change on GitLab.

I've added more commits to armadillo branch 14.2.x. This is to ensure consistency and prevent further potential regressions under gcc 15.

Reply:
@conradsnicta As you may have noticed I have not merged yet.